opm/grid/common/WellConnections.hpp

```diff
@@ -165,7 +165,11 @@ postProcessPartitioningForWells(std::vector<int>& parts,
                                 const WellConnections& well_connections,
                                 std::vector<std::tuple<int,int,char>>& exportList,
                                 std::vector<std::tuple<int,int,char,int>>& importList,
+#if DUNE_VERSION_NEWER(DUNE_GRID, 2, 7)
```

Comment:
I am surprised these parts have been compiling without issue for newer Dune versions. Seems to me there should have been compiler errors before this PR?

Reply:
At least up until recently there were only warnings